package.json

```diff
@@ -155,23 +164,25 @@
     "typescript": "5.0.4",
     "vfile-message": "3.1.4"
   },
-  "resolutions": {
-    "@babel/preset-env": "^7.18.6",
-    "@pmmmwh/react-refresh-webpack-plugin": "0.5.10",
-    "@types/eslint": "^8.2.2",
-    "@types/react": "17.0.56",
-    "@types/react-dom": "17.0.20",
-    "@types/react-router": "5.1.20",
-    "@typescript-eslint/eslint-plugin": "^5.10.0",
-    "@typescript-eslint/parser": "^5.10.0",
-    "core-js-compat": "^3.23.4",
-    "intl-messageformat-parser": "6.4.4",
-    "babel-plugin-typescript-to-proptypes@1.4.2": "patch:babel-plugin-typescript-to-proptypes@npm:1.4.2#.yarn/patches/babel-plugin-typescript-to-proptypes-npm-1.4.2-81920f5edd.patch"
+  "pnpm": {
+    "overrides": {
+      "@types/eslint": "^8.2.2",
+      "@types/react": "17.0.56",
+      "@types/react-dom": "17.0.20",
+      "@types/react-router": "5.1.20",
+      "@typescript-eslint/eslint-plugin": "^5.52.0",
+      "@typescript-eslint/parser": "^5.52.0",
+      "@commercetools-docs/gatsby-theme-docs>gatsby-plugin-mdx": "3.20.0"
+    },
+    "patchedDependencies": {
+      "babel-plugin-typescript-to-proptypes@1.4.2": "patches/babel-plugin-typescript-to-proptypes@1.4.2.patch",
+      "gatsby-plugin-mdx@3.20.0": "patches/gatsby-plugin-mdx@3.20.0.patch"
```

Comment:
Ugly workaround, due to the patch included in the docs-kit packages (based on Yarn patch).

What we're doing here is
1. Using the override `@commercetools-docs/gatsby-theme-docs>gatsby-plugin-mdx` to use the version `3.20.0` (instead of the `patch:` protocol)
2. Patching the `gatsby-plugin-mdx@3.20.0` with the copy of the patch

Essentially we're re-patching the package using pnpm.